backend/de.metas.adempiere.adempiere/serverRoot/de.metas.adempiere.adempiere.serverRoot.base/src/main/java/de/metas/adempiere/scheduler/SchedulerService.java

```diff
@@ -92,7 +99,11 @@ private void handleSupervisor(@NonNull final I_AD_Scheduler scheduler, @NonNull
 		}
 
 		final I_AD_User user = userDAO.getById(supervisorId);
-		user.setIsActive(ManageSchedulerRequest.Advice.ENABLE.equals(supervisorAdvice));
+
+		final boolean activateUser = supervisorAdvice.equals(ManageSchedulerRequest.Advice.ENABLE)
+				|| supervisorAdvice.equals(ManageSchedulerRequest.Advice.RESTART);
+
+		user.setIsActive(activateUser);
```

Comment:
We shouldn't always deactivate the user, because a scheduler might e.g. be linked to the metasfresh user and we need that one to remain active.
Therefore i think we might need to have an explicit "deactivateUserWhenDisablingScheduler" flag which we can then set to true in DisableSchedulerForExternalSystem.java